app/assets/javascripts/templates/modal.jsx

```diff
@@ -0,0 +1,17 @@
+var Modal = ReactBootstrap.Modal;
+
+class DefaultModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
```

Comment:
if it doesn't mess up functionality it might be better to define a default value for showModal here